app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
```

Comment:
The code has a lot of logic in the global scope. This makes maintenance, understanding and testing difficult. Move logic to helper functions. Example: def parse_arguments(args):
    # Logic for processing arguments
    return filename, directories

def collect_file_data():
    # Logic to collect data from the file
    return file_data